tests/test_class2D.py

```diff
@@ -152,6 +180,7 @@ def testIncorrectComponents(self):
                 self.clean_src,
                 self.clean_fspca_basis,  # 400 components
                 fspca_components=100,
+                bispectrum_components=100,
                 large_pca_implementation="legacy",
```

Comment:
Why do we have to add this? Shouldn't this not be needed since 400 > 300 and the bispectrum check comes later anyway?